### PR TITLE
Fyle 1tz94ty : Fixed list element does not have right spacing inside pop ups

### PR DESCRIPTION
**Pull request by siddharth0-0**
image

image

### CODE REVIEW

**Review by Dimple16 — APPROVED** [review]

**Review by Dimple16 — CHANGES_REQUESTED** [review]
include the changes of text alignment

**Comment by siddharth0-0** [comment]
image

File: src/app/shared/components/fy-select/fy-select-modal/fy-select-modal.component.scss [hunk 1]
```diff
@@ -5,8 +5,8 @@ $list-element-color: #333;
 .selection-modal {
   color: black;
   &--list-element {
-    padding: 6px 0;
-    margin: 16px 0px 0px 16px;
+    padding: 18px 0;
+    margin: 0 16px 0 16px;
```
Comment (Dimple16) [inline src/app/shared/components/fy-select/fy-select-modal/fy-select-modal.component.scss:9]: `margin: 0 16px;` should work

Comment (Dimple16) [reply src/app/shared/components/fy-select/fy-select-modal/fy-select-modal.component.scss:9]: refer this - https://developer.mozilla.org/en-US/docs/Web/CSS/Shorthand_properties

**Review by Dimple16 — CHANGES_REQUESTED** [review]
few comments

**Review by Dimple16 — APPROVED** [review]

**Review by Julias0 — APPROVED** [review]

**Pull request merged**